NationStatesAPIBot/NationStatesAPIBot/DumpData/CAUSE.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DumpData
```

Comment:
Could you use something like 'NationStatesAPIBot.DumpData' as Namespace instead?

Reply:
Sure, I see now that that's more consistent with the rest